inclusive-design/idrc
# chore: switch from JSDom to Linkedom (see fluid-project/trivet#115)

## Description

This PR implements the changes from https://github.com/fluid-project/trivet/pull/115/, replacing JSDom with [linkedom](https://www.npmjs.com/package/linkedom) for improved performance of the parse transform.

## Steps to test

1. Check out this branch.
2. Install dependencies: `npm ci`
3. Run a build: `npm run build`

**Expected behavior:** Site builds without issues.

## Additional information

See: https://github.com/fluid-project/trivet/pull/115/

## Related issues

See: https://github.com/fluid-project/trivet/pull/115/
